app/src/pages/inside/productVersionPage/linkedTestCasesTab/tagList/tagList.jsx

```diff
@@ -20,17 +20,19 @@ import classNames from 'classnames/bind';
 import { useIntl } from 'react-intl';
 import { Button } from '@reportportal/ui-kit';
 
+import { isEmpty } from 'lodash.isempty';
```

Comment:
I think this will throw an error

import isEmpty from 'lodash.isempty';

Reply:
fixed